pjmedia/src/pjmedia/stream.c

```diff
@@ -1891,7 +1891,11 @@ static void on_rx_rtp( pjmedia_tp_cb_param *param)
     /* Update RTP session (also checks if RTP session can accept
      * the incoming packet.
      */
-    check_pt = (hdr->pt != stream->rx_event_pt) && PJMEDIA_STREAM_CHECK_RTP_PT;
```

Comment:
doesn't this already mean the same thing as below? i.e. if (PJMEDIA_STREAM_CHECK_RTP_PT==0) check_pt = (... && 0) = 0. And if  (PJMEDIA_STREAM_CHECK_RTP_PT==0) check_pt = (hdr->pt != stream->rx_event_pt) && 1;